src/pages/Home.tsx

```diff
@@ -52,18 +52,19 @@ export const Home: React.FC = () => {
         globalDispatch(initConfig())
             .then(() => console.log('Initialized global state.'))
             .catch((error) => console.error(error));
-    }, []);
+    }, [globalDispatch]);
 
     if (version === undefined || recentProjects === undefined) {
-        return <h1>Loading</h1>
+        return <h1>Loading...</h1>;
     }
 
     return (
         <>
             <div className="container px-20 pt-16 h-screen overflow-clip">
```

Comment:
I feel like with the larger window size, the top could use some more padding.
```suggestion
            <div className="container px-20 pt-24 h-screen overflow-clip">
```